test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs

```diff
@@ -91,7 +92,12 @@ public ErrorPageTests(MvcTestFixture<ErrorPageMiddlewareWebSite.Startup> fixture
             Assert.Contains(expectedCompilationContent, content);
         }
 
-        [Fact]
+        [ConditionalFact]
+        // MVC Generates portable PDBs when compiling Razor views. NET461 does not understand portable PDBs
```

Comment:
Does the Coherence-Signed issue cover reverting this too or do we need a tracking issue?